core/src/env/srml/types.rs

```diff
@@ -44,3 +45,17 @@ impl<'a> From<&'a [u8]> for Address {
 
 /// The default SRML balance type.
 pub type Balance = u64;
+
+/// The default SRML hash type.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Encode, Decode)]
+pub struct Hash([u8; 32]);
```

Comment:
Are `node-runtime` hash types internally `[u8; 32]`?

Reply:
They are H256 and handled this way throughout.

Reply:
then it is perfect